libraries/AP_BattMonitor/AP_BattMonitor_SMBus.h

```diff
@@ -6,11 +6,13 @@
 #include <AP_HAL/I2CDevice.h>
 #include "AP_BattMonitor_Backend.h"
 #include <utility>
+#include <stdio.h>
```

Comment:
Needed?

Reply:
Used, for the `snprintf's` in the serial number, product, device, and manufacturer name functions

Reply:
Moved to `AP_BattMonitor_SMBus.cpp` instead